packages/@sanity/imagetool/src/Resize.js

```diff
@@ -0,0 +1,44 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+
+export default class ImageLoader extends React.Component {
+  static propTypes = {
+    image: PropTypes.instanceOf(HTMLImageElement).isRequired,
+    maxWidth: PropTypes.number.isRequired,
+    maxHeight: PropTypes.number.isRequired,
+    children: PropTypes.func.isRequired
+  }
+
+  getCanvas() {
```

Comment:
Won't this leak a new canvas into the document for every `Resize` instance?

Reply:
Whoops, I had implemented a cWU that removed it, but it got missing in action obviously. Added it back now. Thanks!